scripts/settings.json

```diff
@@ -1,10 +1,10 @@
 {
   "settings":
   {
-    "Raw Census Extract location":  "../data/Scout Census Data/Census 2020 Extract (1).csv",
-    "Scout Census location":  "../data/Scout Census Data/Census 2020 Extract (1) with May 2019 fields.csv",
+    "Raw Census Extract location":  "Scout Census Data/Census 2020 Extract (1).csv",
+    "Scout Census location":  "Scout Census Data/Census 2020 Extract (1) with May 2019 fields.csv",
     "Full ONS PD location":        "../data/ONSPD_MAY_2019_UK/Data/ONSPD_MAY_2019_UK.csv",
-    "Reduced ONS PD location": "../data/ONSPD_MAY_2019_UK/Data/ONSPD_MAY_2019_UK reduced.csv",
+    "Reduced ONS PD location": "ONSPD_MAY_2019_UK/Data/ONSPD_MAY_2019_UK reduced.csv",
     "ONS Names and codes folder": "../data/ONSPD_MAY_2019_UK/Documents/",
```

Comment:
This doesn't look super consistent but I might be missing something. Happy to continue - just keep as an issue (or a new issue) if consistency to be resolved in the future. (I assume the intention is to remove all the ../data parts of the file path).

Reply:
It was relativley easy, so done